doc/classes/PhysicsMaterial.xml

```diff
@@ -14,6 +14,7 @@
 		</member>
 		<member name="bounce" type="float" setter="set_bounce" getter="get_bounce" default="0.0">
 			The body's bounciness. Values range from [code]0[/code] (no bounce) to [code]1[/code] (full bounciness).
+			[b]Note:[/b] Even with [member bounce] set to [code]1.0[/code], some energy will be lost over time due to linear and angular damping. To have a [PhysicsBody3D] that preserves all its energy over time, set [member bounce] to [code]1.0[/code], the body's linear damp mode to Replace (if applicable), its linear damp to [code]0.0[/code], its angular damp mode to Replace (if applicable), and its angular damp to [code]0.0[/code].
```

Comment:
I gather `[constant RigidBody3D.DAMP_MODE_REPLACE]` would have made this note even more verbose than it already is, but at least the quotes:
```suggestion
			[b]Note:[/b] Even with [member bounce] set to [code]1.0[/code], some energy will be lost over time due to linear and angular damping. To have a [PhysicsBody3D] that preserves all its energy over time, set [member bounce] to [code]1.0[/code], the body's linear damp mode to "Replace" (if applicable), its linear damp to [code]0.0[/code], its angular damp mode to "Replace" (if applicable), and its angular damp to [code]0.0[/code].
```

Reply:
I've used bold instead which is what we use to highlight menu options in the editor.